content/articles/ordering-lets-encrypt-certificate.markdown

```diff
@@ -49,26 +49,10 @@ The order is the first step into getting an SSL certificate. It will create an S
 
     1.  [Read this article](/articles/ssl-certificate-names) to determine the appropriate host name of your SSL certificate.
     1.  For different plans, the names available will differ. If you have the ability to select alternate names for your certificate, do so. Otherwise, continue to the next step.
-    1.  Select a Contact from your contact list. The contact information will be used to generate the certificate request (CSR). We will generate a private key that is used for your CSR. Make sure to read our [private key policy](https://dnsimple.com/private-key-policy) before you order your certificate.
     1.  Select whether you would like to automatically renew the certificate. If you do so, the certificate will be renewed 30 days prior to expiration as recommended by Let's Encrypt.
     1.  Submit the order.
 </div>
 
-### What this looks like
-
-As mentioned above, you will see a different form for ordering a Let's Encrypt certificate based on the plan to which you are subscribed. If you have full access to SAN certificate configuration, this is what you'll see:
-
-![Let's Encrypt with SAN](/files/lets-encrypt-new-san.png)
-
-Other plans provide the ability to configure only one name on your Let's Encrypt certificate:
-
-![Let's Encrypt with single name configuration](/files/lets-encrypt-new-names.png)
-
-On some plans we do not allow the configuration of a name but instead configure the certificate to work only on the apex and www-subdomain of the zone:
-
-![Let's Encrypt with fixed names](/files/lets-encrypt-new-fixed.png)
-
-
```

Comment:
I'm doubtful about this removal... I think it is useful so customers can visually tell the difference when setting up LE certs with different name configurations.

Reply:
I'm really skeptical about this... the purpose of the removed section is not clear to me.

If it's about explaining what's the difference between plans regarding ordering LE certs, then I'd argue that the screenshots don't provide any valuable information. A simple bullet-point list would be much more efficient for that.

However, the pricing page does a better job comparing plans... Maybe we could replace the whole section with a message like "Check out our [pricing page](https://dnsimple.com/pricing) to understand limitations on your plan regarding LetsEncrypt certificates"

If we maintain the current section, then we're duplicating information in a way that is hard for us to maintain and (IMO) doesn't provide much value to our customers.

Reply:
🤷‍♂️ I have no strong opinion on this. So if you think it's worth removing, I'm ok with it.